### DISCUSSION TITLE
选中文字创建反链的快捷方式

### DISCUSSION THREAD

**Discussion by leidichen** [obsidianzh/forum#74] [Ideas想法]
- [[ → 可以创建一个反链 ☑
- 当我们选中文字时，按下[[，是否可以将选中文字创建一个反链？🔲
- 当然也可以设置一个快捷键`ctrl+alt+k`，即选中文字创建反链？🔲

**Comment by chen-bao-x** [comment]
希望 Obsidian 早点开放  “用户脚本” 相关的功能和  API，其实这些都挺容易做的。
